### PR TITLE
Migration/chordsymbol

### PR DESCRIPTION
**Pull request by rvilarl**
ChordSymbol!

### CODE REVIEW

**Comment by rvilarl** [comment]
Conflicts resolved

File: src/chordsymbol.ts [hunk 1]
```diff
@@ -715,7 +733,7 @@ export class ChordSymbol extends Modifier {
     } /* CENTER_STEM */ else {
       x = note.getStemX() - this.getWidth() / 2;
     }
-    L('Rendering ChordSymbol: ', this.text, x, y);
+    L('Rendering ChordSymbol: ', this.textFont, x, y);
```
Comment (0xfe) [inline src/chordsymbol.ts:736]: is this intentional?

Comment (rvilarl) [reply src/chordsymbol.ts:736]: Yes, `this.text` was only used in the log function so I thought it must be a bug. `this.textFont` was used overall.

File: src/chordsymbol.ts [hunk 2]
```diff
@@ -13,90 +13,104 @@ import { Flow } from './tables';
...
+  getYOffsetForText(text: string): number {
     let acc = 0;
     let ix = 0;
     const resolution = this.textFont.resolution;
     for (ix = 0; ix < text.length; ++ix) {
       const metric = this.textFont.getMetricForCharacter(text[ix]);
 
       if (metric) {
-        acc = metric.y < acc ? metric.y : acc;
+        acc = metric.y_max < acc ? metric.y_max : acc;
```
[the remark this answers is not in the document]
Comment (rvilarl) [reply src/chordsymbol.ts:113]: That was a bug, I believe. TextFontMetric only have `y_max`, `y_min`.

File: src/chordsymbol.ts [hunk 3]
```diff
@@ -13,90 +13,104 @@ import { Flow } from './tables';
 import { Glyph } from './glyph';
 import { TextFont } from './textfont';
 import { Modifier } from './modifier';
+import { FontInfo } from './types/common';
+import { StemmableNote } from './stemmablenote';
+
+export interface SymbolBlock {
```
[the remark this answers is not in the document]
Comment (rvilarl) [reply src/chordsymbol.ts:19]: Done.

**Comment by rvilarl** [comment]
Ready to review and merge!

**Comment by 0xfe** [comment]
Looks good.

**Pull request merged**